### PR TITLE
Update wgLinterSubmitterWhitelist to mw* and test3 and set wgParsoidSettings

### PR DESCRIPTION
**Pull request by Universal-Omega**
To hopefully fix [T7416](https://phabricator.miraheze.org/T7416).

### CODE REVIEW

**Review by RhinosF1 — APPROVED** [review]
I trust you can copy/paste so not checked the IPs but Thanks!

File: LocalSettings.php [hunk 1]
```diff
@@ -2607,12 +2607,24 @@
 	// Linter
 	'wgLinterSubmitterWhitelist' => [
 		'wmgUseLinter' => [
```
Comment (The-Voidwalker) [inline LocalSettings.php:2609]: Why is this `wmgUseLinter` and not `default`?

Comment (Universal-Omega) [reply LocalSettings.php:2609]: Because that makes it only be used when linter is enabled.

Comment (The-Voidwalker) [reply LocalSettings.php:2609]: Hmm, I'm unfamiliar with this syntax for wgConf, but it does appear that it works.

**Review by The-Voidwalker — APPROVED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (Universal-Omega) [reply LocalSettings.php:2609]: It's something that was added not very long ago.

**Review by Reception123 — APPROVED** [review]

**Pull request merged**